byway/src/main/webapp/index.html

```diff
@@ -3,12 +3,16 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
-    <script src="index.js"></script>
+    <link rel = "stylesheet" href = "index.css">
+    <script src = "index.js"></script>
   </head>
   <body>
-    <div id="content">
-      <div id="title"> byway.</div>
-      <div id="login-link"></div>
+    <div id = "content">
+      <img src = "/images/road.png" id = "road" alt = "road">
```

Comment:
Make these paths absolute (just `images/`) so that if we do restructure our directories, the file can still access these images.

Reply:
done!